src/Sentry.Log4Net/Sentry.Log4Net.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
Unrelated to NLog change. Ideally would go on a separate PR but I'm happy to merge it in.

Reply:
Can revert the change if you want?

Reply:
Please do. If you want to open another PR just for that I'd be happy to merge. Otherwise I'll open one because this was an oversight.

Reply:
Reverted the changes. And forced pushed them away.